### DISCUSSION TITLE
How to preserve inner components of a tab when switching between tabs?

### DISCUSSION THREAD

**Discussion by Fabri19f** [heroui-inc/heroui#2921] [Help]
I'm developing an application that incorporates an extensive form, which has been structured into tabs to improve the navigation experience. For form management and field validation, I have opted to use React Form Hook with the controlled inputs API. However, when switching tabs, the inner components of the previous tab are unmounted, resulting in the loss of validation for the corresponding fields. Is there a recommended way to maintain the inner content of a tab when switching tabs? I would appreciate any advice or suggested approach to resolving this issue.

**Comment by wingkwong** [comment]
related to [this issue](https://github.com/nextui-org/nextui/issues/1562).
